connector/module.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2012-2013 Camptocamp SA
+#    Copyright 2015 anybox SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models
+from openerp.tools.cache import ormcache
+
+
+class IrModuleModule(models.Model):
+    """Overwrite ir.module.module to add cached method 'is_module_installed'.
+    This method is cached, because connector will always check if a module is
+    installed before do action.
+
+    The next methods change the state of the module, then they must invalidate
+    the cache of 'is_module_installed' method:
+
+    * state_update
+    * module_uninstall
+
+    """
+    _inherit = 'ir.module.module'
+
+    @ormcache(skiparg=1)
```

Comment:
Wondering if it shouldn't be:
```
@api.model
@ormcache()
def is_module_installed(self, module_name):
    ....
```
Because the default decorator is `@api.multi`, that's why it you need to set `skiparg=1` where the arg is `ids`, always empty in this case.